examples/dgmulti_2d/elixir_navierstokes_convergence.jl

```diff
@@ -194,7 +194,7 @@ summary_callback = SummaryCallback()
 alive_callback = AliveCallback(alive_interval=10)
 analysis_interval = 100
 analysis_callback = AnalysisCallback(semi, interval=analysis_interval, uEltype=real(dg))
-callbacks = CallbackSet(summary_callback, alive_callback)
+callbacks = CallbackSet(summary_callback, alive_callback, analysis_callback)
 
```

Comment:
This has nothing to do with the $L^1$ errors, but it seems to me that this was missing.